mixins/localize-mixin.js

```diff
@@ -1,14 +1,14 @@
-import {
-	addListener, formatDate, formatDateTime, formatFileSize, formatNumber,
-	formatTime, getDocumentLanguage, getDocumentLanguageFallback, getTimezone,
-	localize, parseDate, parseNumber, parseTime, removeListener
-} from '../helpers/localization.js';
+import {formatDate, formatDateTime, formatTime, parseDate, parseTime} from '@brightspace-ui/intl/lib/dateTime.js';
+import {formatNumber, parseNumber} from '@brightspace-ui/intl/lib/number.js';
+import {formatFileSize} from '@brightspace-ui/intl/lib/fileSize.js';
```

Comment:
Separately, I'd like to go through places that are using these and switch them to just use `intl` directly, leaving this mixin solely for the `localize()` method. Otherwise, any component that just needs to localize a piece of text gets all the date/time/number formatting and parsing included without needing it.

Reply:
Sounds good, should we update the README know to try to stop future use?

Reply:
I'd like to actually go through and remove this from here if it ends up being easy to clean up places that are using it, making a README update unnecessary. If it ends up being too much work and I need to create a new slimmer localize mixin, then yeah I'll deprecate and add a note.